src/write.c

```diff
@@ -876,7 +878,8 @@ static avifResult avifEncoderAddImageInternal(avifEncoder * encoder,
         // Another frame in an image sequence
 
         const avifImage * imageMetadata = encoder->data->imageMetadata;
-        // If the first image had an alpha plane (even if fully opaque), all subsequent images must have alpha as well.
+        // If the first image in the sequence had an alpha plane (even if fully opaque), all
+        // subsequent images must have alpha as well.
```

Comment:
I added back "in the sequence" to this comment. Please let me know if you intended to delete it.

Reply:
Let's add it back for now. I deleted it to cover layer image case, but now I think we should describe it clearly instead.

Reply:
Thank you for the reply. Let's update this comment and the comment at line 878 above when you add support for layered images.